There is a test that fails:

```
exec ${PAGER:-/usr/bin/less} "$0" || exit 1
-----------------------------------------------------------------------------
/b/s/w/ir/run/bazel-out/k8-opt/bin/tensorflow/python/kernel_tests/matmul_op_test.runfiles/org_tensorflow/tensorflow/python/kernel_tests/matmul_op_test.py:231: ComplexWarning: Casting complex values to real discards the imaginary part
  m * k).astype(dtype).reshape([m, k])
/b/s/w/ir/run/bazel-out/k8-opt/bin/tensorflow/python/kernel_tests/matmul_op_test.runfiles/org_tensorflow/tensorflow/python/kernel_tests/matmul_op_test.py:235: ComplexWarning: Casting complex values to real discards the imaginary part
  k * n).astype(dtype).reshape([k, n])
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss2018-06-06 17:48:56.892095: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX
.........................ssssssssssss........................ssssssssssssF.....................................................................................................................................
======================================================================
FAIL: testTransposedStatistics (__main__.MatMulStatsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/b/s/w/ir/run/bazel-out/k8-opt/bin/tensorflow/python/kernel_tests/matmul_op_test.runfiles/org_tensorflow/tensorflow/python/kernel_tests/matmul_op_test.py", line 156, in testTransposedStatistics
    self.assertEqual(7200, flops)
AssertionError: 7200 != 6975
----------------------------------------------------------------------
Ran 268 tests in 3.490s
FAILED (failures=1, skipped=85)
```

--- a/tensorflow/python/kernel_tests/matmul_op_test.py
+++ b/tensorflow/python/kernel_tests/matmul_op_test.py
@@ -142,7 +142,7 @@ class MatMulStatsTest(test_lib.TestCase):
       for op in g.get_operations():
         flops = ops.get_stats_for_node_def(g, op.node_def, "flops").value
         if op.name == "MatMul":
-          self.assertEqual(7200, flops)
+          self.assertEqual(6975, flops)
 
   def testTransposedStatistics(self):
     g = ops.Graph()
@@ -153,7 +153,7 @@ class MatMulStatsTest(test_lib.TestCase):
       for op in g.get_operations():
         flops = ops.get_stats_for_node_def(g, op.node_def, "flops").value
         if op.name == "MatMul":
-          self.assertEqual(7200, flops)
+          self.assertEqual(6975, flops)
 
 
 try:

--- a/tensorflow/python/ops/math_ops.py
+++ b/tensorflow/python/ops/math_ops.py
@@ -2033,7 +2033,7 @@ def _calc_mat_mul_flops(graph, node):
   output_shape = graph_util.tensor_shape_from_node_def_name(graph, node.name)
   output_shape.assert_is_fully_defined()
   output_count = np.prod(output_shape.as_list())
-  return ops.OpStats("flops", (k * output_count * 2))
+  return ops.OpStats("flops", ((2 * k - 1) * output_count))
 
 
 def _as_indexed_slices(x, optimize=True):

